Rmd/DESeq2_report_new.Rmd

```diff
@@ -14,7 +14,7 @@ params:
   species: "human"                 # one of human, mouse, rat, hamster
   design: "group"                # single experimental group of interest; entries in this column must match the contrast names.
   intgroup: ["group"]            # experimental group of interest plus covariates; can be more than one
-  intgroup_to_plot: "group" # for PCA plots, add tabs for one or more groups to color
+  intgroup_to_plot: ["group"]   # for PCA plots, add tabs for one or more groups to color
```

Comment:
So this should be able to handle a vector of length greater than one right? I thought I tried and it didn't seem to work for me.

Reply:
It used to work fine, yeah. I thought this was just something that got mangled in the rebase from your changes. I will make sure this is working before merge.